app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -332,11 +332,12 @@ export default class SampleViewV2 extends React.Component {
 
     // taxon's category was selected and its subcategories were not excluded
```

Comment:
so categories is filter by inclusion and subcategories by exclusion? seems confusing!

Reply:
I think subcategories are also filtered by inclusion; there's a condition just above this one that explicitly checks the subcategory.